### DISCUSSION TITLE
Libusb error when scaning or connecting crazyfile on cfclient

### DISCUSSION THREAD

**Discussion by fairwel1** [bitcraze/discussions#1681] [Q&A] [answered]
hellow everyone, 

I am a new crazyfile user and i have installed cfclient on windows.There is something error happening when scaning crazyfile on cfclient.
INFO:cfclient.utils.input.inputreaders.pysdl2:Looking for devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Found 0 devices
[Errno None] b'libusb0-dll:err [control_msg] sending control message failed, win error: \xd0\xc5\xba\xc5\xb5\xc6\xb3\xac\xca\xb1\xca\xb1\xbc\xe4\xd2\xd1\xb5\xbd\r\n\n'
Maybe the crazyradio is installed successfully because i can see it blinking red when launching the cfclient.  
![image](https://github.com/user-attachments/assets/4b617327-0850-4dfa-8106-8996a4e3cd4d)
![image](https://github.com/user-attachments/assets/acde71c6-2c98-4966-9bbb-e7c952e8d8a4)

**Comment by fairwel1** [comment]
Another error when scaning  successfully. It seems like a same error as metioned above.
![image](https://github.com/user-attachments/assets/547f155c-546c-4e9b-ba68-1f8aaed4e75e)

**Reply by Bananazooka** [reply]
Hi, 
Have you installed the drivers for the Crazyradio? You'll find instructions [here](https://www.bitcraze.io/documentation/repository/crazyradio-firmware/master/building/usbwindows/)

**Reply by fairwel1** [reply, marked as answer]
Hi,
I have solved the problem by changing my crazyradio firmware from 1.2 to 1.1
